client/src/components/AllUsers.js

```diff
@@ -1,129 +1,153 @@
-		{trainees.length <=1 ? (<div>
-			<ResultPage filter={setFilter}/>
-		</div>) :(<div>
-			<h1 style={{ color: "yellow", textAlign: "center" }}>Trainees with matching availability</h1>
-			<div className="container">
-				<div className="row">
-					<div className="col-md-6">
-						<input
-							type="text"
-							className="form-control"
-							placeholder="Search"
-							value={search}
-							onChange={handleSearch}
-						/>
-					</div>
-					<div className="col-md-6">
-						<div className="btn-group" role="group" aria-label="Basic example">
-							<button
-								type="button"
-								className="btn btn-outline-info"
-								onClick={handleDailyFilter}
-							>
-								Daily
-							</button>
-							<button
-								type="button"
-								className="btn btn-outline-info"
-								onClick={handleWeeklyFilter}
-							>
-								Weekly
-							</button>
-							<button
-								type="button"
-								className="btn btn-outline-info"
-								onClick={handleMonthlyFilter}
-							>
-								Monthly
-							</button>
-						</div>
-					</div>
-				</div>
-			</div>
-			<div>
-				<span>
-					<div className="col-md-6">
-						<div className="btn-group" role="group" aria-label="Basic example">
-						</div>
-					</div>
-				</span>
-			</div>
-			<div className="container">
-				<div className="row">
-					{trainees.map((trainee) => {
-						return (
-							<div className="col-md-6">
-								<div
-									className="card mb-4 shadow-sm"
-									style={{
-										background: "#2f4d0a",
-										border: "black solid 1px",
-										margin: "2px",
-										padding: "2px",
-									}}
-								>
-								<AvailabilityCards
-										key={trainee.id}
-										trainee={trainee}
-										trainees={trainees}
-										setTrainees={setTrainees}
-									/>
-								</div>
-							</div>
-						);
-					})}
-				</div>
-			</div>
-		</div>)}
-		</div>
-	);
+  return (
+    <div>
+      <div>
+        <NavBar />
+      </div>
+      {trainees.length <= 1 ? (
+        <div>
+          <ResultPage filter={setFilter} />
+        </div>
+      ) : (
+        <div>
+          <h1
+            style={{
+              fontFamily: "sen",
+              color: "black",
+              textAlign: "center",
+              marginTop: "100px",
+              marginBottom: "50px",
+            }}
+          >
+            Trainees with <br /> matching availability
+          </h1>
+          <div className="container">
+            <div className="row">
+              <div className="col-md-6">
+                <input
+                  type="text"
+                  className="form-control"
+                  placeholder="Search"
+                  value={search}
+                  onChange={handleSearch}
+                />
+              </div>
+              <div className="col-md-6">
+                <div
+                  className="btn-group"
+                  role="group"
+                  aria-label="Basic example"
```

Comment:
It's good to see accesibility lables but should be updated to reflect what this btn group is.
For example "Buttons to filter results"

Reply:
Thank you for spotting that, it's now updated.